### PR TITLE
README upgrades

### PR DESCRIPTION
**Pull request by jason-curtis**
- clearer instructions
- pip install actually works and is --quiet

### CODE REVIEW

File: README.md
Comment (dubchuck) [inline README.md:48]: Should we inform users to add a tag or another label so that feature requests are clearly for notebooks?

File: README.md
Comment (dubchuck) [inline README.md:58]: Perhaps a hashed link that jumps to the Python section?  I can see the value of needing to look at the overview before getting there.

**Review by dubchuck — APPROVED** [review]
A few questions but overall looks good!

File: .github/CODEOWNERS
Comment (jaimemarijke) [inline .github/CODEOWNERS:3]: Did you also add the restriction that commits to master must be pull requests? And that merges require approval from a codeowner?

**Review by jaimemarijke — APPROVED** [review]

File: README.md
Comment (jason-curtis) [reply README.md:58]: there isn't really just one python section that covers everything. a bunch of the general stuff applies too.

File: README.md
Comment (jason-curtis) [reply README.md:48]: let's see if there's any confusion first

File: .github/CODEOWNERS
Comment (jason-curtis) [reply .github/CODEOWNERS:3]: thanks for the reminder, just did it

**Pull request merged**
